platform/darwin/src/MGLLoggingConfiguration_Private.h

```diff
@@ -1,5 +1,9 @@
 #import "MGLLoggingConfiguration.h"
 
+NS_INLINE NSString *NSStringFromBOOL(BOOL value) {
```

Comment:
Functions defined by this library should be prefixed with the library’s class prefix `MGL`, not Foundation’s prefix `NS`.